django_declarative_apis/machinery/filtering.py

```diff
@@ -67,6 +68,34 @@ def _get_unexpanded_field_value(inst, field_name, field_type):
         return {display_key: getattr(obj, display_key)}
 
 
+def _is_relation(field):
+    return field.is_relation or getattr(field, "is_fake_relation", False)
```

Comment:
this adds support for fake relationships. In example `CharField`s acting as foreign keys.